### DISCUSSION TITLE
Car Wars

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#374] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/car-wars/

**Comment by brunoaugusto** [comment]
Não sendo ruim como "Aviões" foi, tudo bem

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/car-wars/#comment_1087485084>):

> Não sendo ruim como "Aviões" foi, tudo bem

É... Aviões foi de cair o c\* da bunda

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/car-wars/#comment_1087485084>):

> Não sendo ruim como "Aviões" foi, tudo bem

Verdade, e olha que já assisti a cada porcaria...
